plugins/hazelcast37/src/main/java/org/radargun/service/Hazelcast37ClientService.java

```diff
@@ -0,0 +1,57 @@
+package org.radargun.service;
+
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.core.MemberAttributeEvent;
+import com.hazelcast.core.MembershipEvent;
+import com.hazelcast.core.MembershipListener;
+import org.radargun.Service;
+import org.radargun.config.Property;
+import org.radargun.traits.ProvidesTrait;
+
+/**
+ * Hazelcast client service
+ *
+ * @author Roman Macor &lt;rmacor@redhat.com&gt;
+ */
+
+@Service(doc = "Hazelcast client")
+public class Hazelcast37ClientService extends Hazelcast36Service {
```

Comment:
Do the clients use the same API as library mode, and are the capabilities same? If it's the same and the only difference is configuration, you don't need separate service, just add the config options to Hazelcast36Service itself.